sdk/storage/azure-storage-files-shares/inc/azure/storage/files/shares/protocol/share_rest_client.hpp

```diff
@@ -342,7 +342,7 @@ namespace Azure { namespace Storage { namespace Files { namespace Shares {
   {
     std::string LastModified;
     std::string Etag;
-    int32_t Quota = int32_t();
+    int64_t Quota = int64_t();
```

Comment:
Why you changed the other to `QuotaInGiB` while keep this unchanged?

Reply:
This is generated by swagger and swagger is calling it Quota. It is not visible to customers as well.